docs/README.md

```diff
@@ -0,0 +1,91 @@
+---
+sort: 0
+weight: 0
+title: VictoriaMetrics Operator
+---
+
+# VictoriaMetrics Operator
+
+Operator serves to make running VictoriaMetrics applications on top of Kubernetes as easy as possible while preserving Kubernetes-native configuration options.
+
+VictoriaMetrics Operator (`vmoperator`) is the classic kubernetes-operator for VictoriaMetrics with many [great features](#features).
+It allows you to manage Victoria Metrics components in Kubernetes or OpenShift clusters
+in a declarative style according to [GitOps](https://www.redhat.com/en/topics/devops/what-is-gitops)
+and [IaC](https://en.wikipedia.org/wiki/Infrastructure_as_code) concepts.
+
+VictoriaMetrics also provides [helm charts](https://github.com/VictoriaMetrics/helm-charts) without operator.
+Operator makes the same, simplifies it and provides [advanced features](https://docs.victorimetrics.com/operator/#features).
+
+Learn more about [key concepts](#key-concepts) of `vmoperator` and follow the **[quick start guide](https://docs.victoriametrics.com/operator/quickstart.html)** for a better experience.
+
+## Features of vmoperator
+
+- Deployment and management in a kubernetes clusters of any number of VictoriaMetrics applications (like vmsingle/vmcluster instances and another components like vmauth, vmagent, vmalert, etc...)
+- Seamless [migration from prometheus-operator](https://docs.victoriametrics.com/operator/migration.html) with auto-conversion of prometheus [custom resources](#custom-resources)
```

Comment:
can we use relative links? this simplify the maintability of link for docs, new docs and operator repo